doc/articles/getting-started/wizard/includes/extensions.md

```diff
@@ -47,9 +47,9 @@ dotnet new unoapp -http
 The Localization extension is responsible for managing globalization in your app.  
 This enables keeping all translations of your app in a single place and enables the user to easily switch the UI language.
 
-The generated app contains *resw* files which can be used to define the application strings. The defined languages are configured in *appsettings.json*. Follow [this link](Learn.Tutorials.Localization.HowToUseLocalization#3-update-the-ui-culture-with-localizationsettings) to learn how to switch the UI culture.
+The generated app contains *resw* files which can be used to define the application strings. The defined languages are configured in *appsettings.json*. Follow [this link](xref:Learn.Tutorials.Localization.HowToUseLocalization#3-update-the-ui-culture-with-localizationsettings) to learn how to switch the UI culture.
```

Comment:
Can you confirm on staging if this link work?

The xref I see in uno.extensions repo is `Uno.Extensions.Localization.HowToUseLocalization`: https://github.com/unoplatform/uno.extensions/blob/main/doc/Learn/Localization/HowTo-Localization.md (looks like it broke by https://github.com/unoplatform/uno.extensions/commit/44fc24911c4265c71707e09185aee8d3dd943b5c)

Reply:
@Youssef1313 Hmmm... not able to confirm as all the xref links on the staging site doesn't seem to actually do anything:  

https://github.com/user-attachments/assets/7c059cda-aa99-46a1-b635-7e0bc2ece270

Reply:
I updated the link to the one you suggested and adjusted the anchor as well, now it should work in prod.